test/scripts/docker-compose.yml

```diff
@@ -17,11 +16,10 @@ services:
     ports:
       - "4566-4599:4566-4599"
     volumes:
-      - '${TMPDIR:-/tmp/localstack}:/tmp/localstack'
       - '/var/run/docker.sock:/var/run/docker.sock'
-      - ./localstack:/docker-entrypoint-initaws.d
+      - ./localstack:/etc/localstack/init/ready.d
     healthcheck:
-      test: curl http://localhost:4566/health || exit 1
+      test: curl http://localhost:4566/_localstack/health || exit 1
```

Comment:
I'll need to read up on these changes and the removal of `command`

Reply:
Not sure what that command block was doing anyway; perhaps you can explain a bit?
Localstack launches as expected and runs the init.sh hook mounted in /etc/localstack/init/ready.d; is there anything else that needs to happen before running the integration tests?

Reply:
Honestly I can't remember 😆

Reply:
I've updated the healthcheck CMD to be a bit nicer; it checks if SQS service specifically is in 'running' now.

Reply:
Seems to still spin up the docker, so this appears to be fine.